### PR TITLE
IRO-1403: Show if Block is on Main Chain or Fork

### PR DESCRIPTION
**Pull request by yuryironfish**

### CODE REVIEW

**Comment by NullSoldier** [comment]
Does this follow the design? I don't see in figma that it should show a tags on the main block. Is there a discussion you had with Skylar around this? <https://www.figma.com/file/WOuM3Pvq0QhIvWRBr0EvHO/Block-Explorer?node-id=1401%3A9293>

File: src/container/BlockDetailPage/BlockDetailPage.tsx [hunk 1]
```diff
@@ -69,10 +69,21 @@ const BlockDetailPage = () => {
...
       />
-
       <BoxWrapper
         isLoading={service.status === ServiceState.LOADING}
-        header={t('app.blockDetailPage.information.title')}
+        header={
+          main ? (
+            <>
+              {t('app.blockDetailPage.information.title')}{' '}
+              <SmallChip text={t('app.blockDetailPage.information.main')}></SmallChip>
```
Comment (NullSoldier) [inline src/container/BlockDetailPage/BlockDetailPage.tsx:78]: Does this follow the design? I don't see in figma that it should show a tags on the main block. Is there a discussion you had with Skylar around this? <https://www.figma.com/file/WOuM3Pvq0QhIvWRBr0EvHO/Block-Explorer?node-id=1401%3A9293>

File: src/container/BlockDetailPage/BlockDetailPage.tsx [hunk 2]
```diff
@@ -69,10 +69,18 @@ const BlockDetailPage = () => {
...
-
       <BoxWrapper
         isLoading={service.status === ServiceState.LOADING}
-        header={t('app.blockDetailPage.information.title')}
+        header={
+          main ? (
+            <>{t('app.blockDetailPage.information.title')}</>
+          ) : (
+            <>
+              {t('app.blockDetailPage.information.title')}{' '}
```
Comment (NullSoldier) [inline src/container/BlockDetailPage/BlockDetailPage.tsx:79]: Can we not duplicate the title? I think this looks a bit odd.

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -93,6 +93,8 @@
       },
       "information": {
         "title": "Block Information",
+        "main": "Main",
```
Comment (NullSoldier) [inline public/locales/en/translation.json:96]: Is this used anywhere?

File: src/container/BlockDetailPage/BlockDetailPage.tsx [hunk 4]
```diff
@@ -69,10 +69,18 @@ const BlockDetailPage = () => {
...
         isLoading={service.status === ServiceState.LOADING}
-        header={t('app.blockDetailPage.information.title')}
+        header={
+          <>
+            {t('app.blockDetailPage.information.title')}{' '}
+            {!main ? (
+              <SmallChip text={t('app.blockDetailPage.information.forked')}></SmallChip>
+            ) : (
+              ''
+            )}
```
Comment (NullSoldier) [inline src/container/BlockDetailPage/BlockDetailPage.tsx:81]: You can also conditionally render elements using this syntax

```jsx
{main && <>Hello!</>}
```

Comment (yuryironfish) [reply src/container/BlockDetailPage/BlockDetailPage.tsx:79]: Updated.

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**